lib/marc_to_argot/macros/shared/publication_year.rb

```diff
@@ -27,7 +27,7 @@ def publication_year(options = {})
           lambda do |rec, acc|
             ff_date_type = rec.date_type
             date = set_year_from_008(rec, ff_date_type, min_year, max_year, range_tolerance) if field_present?(rec, '008')
-            date = set_year_from_varfield(rec, ff_date_type, min_year, max_year, range_tolerance) if date == nil
+            date = set_year_from_varfield(rec, ff_date_type ||= 's', min_year, max_year, range_tolerance) if date == nil
```

Comment:
If `s` is intended as the default value for `ff_date_type` I would probably set it as part of the method definition instead (note that args with defaults must be last). So on line 76:

`def set_year_from_varfield(rec, min, max, range_tolerance, ff_date_type = 's')`

Reply:
Changed.